src/main/java/baseball/domain/Answer.java

```diff
@@ -0,0 +1,58 @@
+package baseball.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static baseball.utils.ExceptionMessage.*;
+
+public class Answer {
+
+    private static final int ANSWER_DEFAULT_SIZE = 3;
+    private static final int MIN_BALL_SIZE = 1;
+    private static final int MAX_BALL_SIZE = 9;
+    private final List<Integer> numbers;
```

Comment:
상수와 필드 사이에 한칸 공백을 두면 더 읽기 수월할 것 같네요
```suggestion
    private static final int ANSWER_DEFAULT_SIZE = 3;
    private static final int MIN_BALL_SIZE = 1;
    private static final int MAX_BALL_SIZE = 9;
    
    private final List<Integer> numbers;
```

Reply:
넵!!!